src/charm.py

```diff
@@ -121,7 +147,7 @@ def flask_layer(self) -> dict:
                 "flask-app": {
                     "override": "replace",
                     "summary": "Flask application service",
-                    "command": self._webserver.command,
+                    "command": shlex.join(self._webserver.command),
```

Comment:
I think it would be more cohesive to return the string resulting from `shlex.join` in `self._webserver.command`

Reply:
Yes, there's another method called `GunicornWebserver.check_config_command` that is used by `ops.model.Container.exec`, which requires a list as its argument. To maintain consistency between the `check_config_command` and `command` methods, I decided to let both of them to return a list of strings. Since converting from a list of strings to a single string is simpler than the other way around.

Reply:
Understood. Thx